parkings/tests/test_fix_geometries.py

```diff
@@ -0,0 +1,143 @@
+import os
+
+import pytest
+from django.core import management
+
+from parkings.management.commands import fix_geometries
+
+from ..models import ParkingArea
+from .utils import call_mgmt_cmd_with_output
+
+directory = os.path.abspath(os.path.dirname(__file__))
+
+parking_areas_yaml = os.path.join(directory, 'invalid_geom_parking_areas.yaml')
+
+
+def test_non_compatible_model():
+    with pytest.raises(SystemExit) as excinfo:
+        call_the_command('Operator')
+    assert str(excinfo.value) == (
+        'Model should have exactly one MultiPolygon field')
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize('verbosity', [0, 1, 2])
+@pytest.mark.parametrize('dry_run', [False, True])
+def test_fixing(verbosity, dry_run):
+    # Load the test data and collect its timestamps
+    management.call_command('loaddata', parking_areas_yaml)
+    timestamps_before = sorted(
+        ParkingArea.objects.values_list('modified_at', flat=True))
+
+    areas_before = get_areas(ParkingArea.objects.all())
+
+    # Call the command
+    (stdout, stderr) = call_the_command(
+        'ParkingArea', verbosity=verbosity, dry_run=dry_run)
+
+    # Check the output
+    assert stdout == get_expected_output(verbosity, dry_run)
+    assert stderr == ''
+
+    # Check that it did changes as expected
+    timestamps_after = sorted(
+        ParkingArea.objects.values_list('modified_at', flat=True))
+    if dry_run:
+        assert timestamps_after == timestamps_before
+    else:
+        assert timestamps_after != timestamps_before
+
+    areas_after = get_areas(ParkingArea.objects.all())
+
+    assert areas_after == areas_before, "Areas (m^2) shouldn't change"
```

Comment:
Are the actual area sizes floats? If they are, I don't think they should be compared like this, at least in theory :)

Reply:
Yes, they are floats. You're right, maybe I should convert them to Decimals or ints after the rounding.